tests/virt_autotest/esxi_open_vm_tools.pm

```diff
@@ -324,19 +318,6 @@ sub do_clock_sync_tests {
     } else {
         die 'Disabling clock synchronization failed.';
     }
-
-    if (is_qemu) {
-        record_info('Revert VM timesync configs');
-        $powerops = 'power.shutdown';
-        $powerops_ret = take_vm_power_ops($vm_id, $powerops, $VM_POWER_ON);
-        check_vm_power_state($vm_id, $vm_ip, $powerops, $powerops_ret, 0, $VM_POWER_OFF);
-
-        revert_vm_timesync_setting($vm_name);
-
-        $powerops = 'power.on';
-        $powerops_ret = take_vm_power_ops($vm_id, $powerops, $VM_POWER_OFF);
-        check_vm_power_state($vm_id, $vm_ip, $powerops, $powerops_ret, 1, $VM_POWER_ON);
-    }
 }
```

Comment:
Why disable time/clock sync/test completely ?

Reply:
This code change does not disable the test, just using vm tools cmd for instead.